Tests/FileChangeCheckerTests.cs

```diff
@@ -101,8 +87,76 @@ public void ProjectIsModifiedWithDifferentFileOrder()
             mock.Setup(w => w.ReturnOriginalPathFileStream()).Returns(fakeList);
             mock.Setup(w => w.ReturnModifiedPathFileStream()).Returns(reverseFakeList);
 
-            FileChangeChecker.ProjectIsModified(fakeDir).Should().BeFalse();
+            new FileChangeChecker(fakeDir).FileChangeEvidence.EvidencePresent.Should().BeFalse();
+
+        }
+
+        [Fact]
+        public void FileChangeChecker_FeatureEvidence_EvidencePresentFieldTrue()
+        {
+            var fakeDirectory = mock.Object;
+
+            List<FileContent> fakeList = new List<FileContent>();
+
+            List<FileContent> fakeList2 = new List<FileContent>();
+
+            for (int i = 0; i < fakePaths1.Count; i++)
+            {
+                fakeList.Add(CreateFakeStream(i));
+                fakeList2.Add(CreateFakeStream(i));
+
+            }
+            fakeList2.Add(CreateFakeStream(10));
+            mock.Setup(w => w.OriginalPaths).Returns(fakePaths1);
+            mock.Setup(w => w.ModifiedPaths).Returns(fakePaths1);
+
+            fakeList2.RemoveAt(0);
+
+            mock.Setup(w => w.ReturnOriginalPathFileStream()).Returns(fakeList);
+            mock.Setup(w => w.ReturnModifiedPathFileStream()).Returns(fakeList2);
+
+            var fileCheck = new FileChangeChecker(fakeDirectory);
+            fileCheck.FileChangeEvidence.EvidencePresent.Should().Be(true);
+        }
+
+        [Fact]
+        public void FileChangeChecker_FeatureEvidence_EvidenceFieldTrue()
```

Comment:
This test and the test at line 95 have a lot of code duplication, maybe extract a method to set up this same state?

Reply:
True

Reply:
It's actually quite difficult to tell what these tests are testing. I do think we need some refactoring so it is clearer what the purpose of each test it and to remove some of the duplicated setup code.